### ISSUE TITLE
[MISSING] SUBMIT/PUBLISH DIRECT TO SUBSCRIPTION TIERS

### ISSUE THREAD

**Issue by xedis** [wix-incubator/DeviantArt-API#235]
This has a business value add folks. Any chance of getting some resources allocated to extending the API to enable a user to publish to their Sub Tiers. 

Currently I'm forced to push to a premium gallery and then move them 24 at a time into a big bucket  (since thats all there is for each tier) .. Yes i've built out Non-Premium categorized galleries but this is still alot of manual effort. Be nice to be able to push direct to where I want to. Maybe give the sub tiers alias galleryid's that can be used with the existing api functionality?

**Comment by Delavestra** [comment]
Yes, this. I require the ability to use API to publish to subscription tiers. Not having this capability is quite frustrating. Is there any feedback that this is in the works or that we're get this soon? I can't imagine this being difficult to implement... Fingers crossed.

API FOR SUBSCRIBER TIERS PLEASE!
